.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please do not include IDE files in commits

Reply:
IDE files removed from commit. Include .vscode/ in gitignore